Dockerfile

```diff
@@ -0,0 +1,32 @@
+#
+# Yo dawg, I heard you like Docker
+#
+# To build dockerized-classtime: 
+#   docker build -t rosshamish/classtime:latest . 
+#
+# To run dockerized-classtime, simply do
+# 	docker run rosshamish/classtime
+#
+# You may need to do some network configuration to
+# expose port 5000, especially under boot2docker.
+# (documentation on this coming soon? :) )
+#
```

Comment:
I'm in favor of this type of documentation staying in the docs themselves, except perhaps "``You may need to do some network configuration to expose port 5000, especially under boot2docker``".